drivers/comparator/Kconfig.stm32_comp

```
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config COMPARATOR_STM32_COMP
+	bool "ST STM32 comparator driver"
+	default y
+	depends on DT_HAS_ST_STM32_COMP_ENABLED
+	select PINCTRL
+	select USE_STM32_LL_EXTI  # EXTI_STM32
+	select USE_STM32_LL_COMP
+
+if COMPARATOR_STM32_COMP
+
+config COMPARATOR_STM32_COMP_MILLER_EFFECT_HANDLING
+	bool "STM32 Comparator Miller effect handling"
```

```suggestion
	bool
```

Providing a prompt ("[...]") makes the symbol user cofigurable, which we don't want because it only depends on SoC.

---

done